compiler/frontend/src/org/jetbrains/kotlin/diagnostics/Errors.java

```diff
@@ -957,6 +957,9 @@
     DiagnosticFactory1<PsiElement, KtKeywordToken> VAL_OR_VAR_ON_CATCH_PARAMETER = DiagnosticFactory1.create(ERROR);
     DiagnosticFactory1<PsiElement, KtKeywordToken> VAL_OR_VAR_ON_SECONDARY_CONSTRUCTOR_PARAMETER = DiagnosticFactory1.create(ERROR);
 
+    // If expressions
+    DiagnosticFactory0<KtIfExpression> COMMENT_OUT_OF_IF = DiagnosticFactory0.create(ERROR);
```

Comment:
No, you should not create any new compiler diagnostics, you should just modify intention `AddBracesIntention`